Dockerfile

```diff
@@ -1,58 +1,58 @@
-FROM ubuntu:15.04
+FROM alpine:3.5
 MAINTAINER "The Impact Bot" <technology@bcorporation.net>
 
-RUN apt-get update -y && \
-    apt-get install -y \
-    git \
+RUN set -ex && \
+    apk upgrade --update && \
+    apk add --update \
     php5 \
     php5-fpm \
-    php5-curl \
-    php5-gd \
-    php5-geoip \
-    php5-imagick \
-    php5-imap \
     php5-json \
-    php5-ldap \
     php5-mcrypt \
-    php5-mongo \
-    php5-mysqlnd \
     php5-pgsql \
-    php5-redis \
-    php5-sqlite \
-    php5-xdebug \
-    php5-memcached \
-    php5-dev
-
-RUN sed -i "s/;date.timezone =.*/date.timezone = GMT/" /etc/php5/fpm/php.ini && \
-    sed -i "s/;cgi.fix_pathinfo=1/cgi.fix_pathinfo=0/" /etc/php5/fpm/php.ini && \
-    sed -i "s/display_errors = Off/display_errors = stderr/" /etc/php5/fpm/php.ini && \
-    sed -i "s/upload_max_filesize = 2M/upload_max_filesize = 128M/" /etc/php5/fpm/php.ini && \
-    sed -i "s/;opcache.enable=0/opcache.enable=1/" /etc/php5/fpm/php.ini && \
-    sed -i -e "s/;daemonize\s*=\s*yes/daemonize = no/g" /etc/php5/fpm/php-fpm.conf && \
-    echo "clear_env = no" >> /etc/php5/fpm/php-fpm.conf && \
-    sed -i '/^listen = /clisten = 9000' /etc/php5/fpm/pool.d/www.conf && \
-    sed -i '/^listen.allowed_clients/c;listen.allowed_clients =' /etc/php5/fpm/pool.d/www.conf && \
-    sed -i '/^;catch_workers_output/ccatch_workers_output = yes' /etc/php5/fpm/pool.d/www.conf && \
-    sed -i '/^;env\[TEMP\] = .*/aenv[MYSQL_PORT_3306_TCP_ADDR] = $MYSQL_PORT_3306_TCP_ADDR' /etc/php5/fpm/pool.d/www.conf
-
-RUN php5enmod mcrypt
+    php5-pdo_pgsql \
+    php5-xdebug \ 
+    shadow \
+    openssl
+
+RUN apk --no-cache add ca-certificates && \
+    wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://raw.githubusercontent.com/sgerrand/alpine-pkg-php5-memcached/master/sgerrand.rsa.pub && \
+    wget https://github.com/sgerrand/alpine-pkg-php5-memcached/releases/download/2.2.0-r0/php5-memcached-2.2.0-r0.apk && \
+    apk add --allow-untrusted --no-cache ./php5-memcached-2.2.0-r0.apk && \
+    rm php5-memcached-2.2.0-r0.apk && \
+    rm -rf /var/cache/apk/*
+
+RUN groupadd www-data && \
+    useradd -M -u 1000 -o www-data 
+
+RUN sed -i 's|^;date.timezone =.*|date.timezone = America/New_York|' /etc/php5/php.ini && \
```

Comment:
I found the the php-fpm confs didn't override pools the way that that you's think, so I didn't trust the ini files to do the right thing either.  I think you're right though, we could should be able to pull these out into a separate .ini file.  It's kind of nice having a single php.ini as the source of truth though too.